contrib/metrics-exporter/README.md

```diff
@@ -0,0 +1,36 @@
+
+
+Proper Dockerfile for application
+
+```Dockerfile
```

Comment:
this seems prone to skewing from the Dockerfile in the same directory. Can we link instead?